comictalker/comictalker.py

```diff
@@ -130,7 +130,7 @@ def parse_settings(self, settings: dict[str, Any]) -> dict[str, Any]:
         settings is a dictionary of settings defined in register_settings.
         It is only guaranteed that the settings defined in register_settings will be present.
         """
-        if settings[f"{self.id}_key"]:
+        if settings.get(f"{self.id}_key"):
             self.api_key = settings[f"{self.id}_key"]
         if settings[f"{self.id}_url"]:
```

Comment:
Lets add the same fix here

Reply:
Added URL. My thinking was the URL is always needed but it is possible the talker wouldn't want to expose the option?

Reply:
Someone could come up with something that doesn't use a url. Does it make sense? No. but the world doesn't make sense and this is a 5 character fix that doesn't change the standard usecase.

Reply:
Being able to not have a url does indeed make sense as I'm throwing together a talker to use the GCD dump (converted to sqlite) right now.